src/app/organization/holidays/view-holidays/view-holidays.component.html

```diff
@@ -0,0 +1,61 @@
+<div fxLayout="row" fxLayoutAlign="end" fxLayoutGap="2%" fxLayout.lt-md="column" class="container m-b-20">
+  <button mat-raised-button color="primary" (click)="activateHoliday()">
+    Activate
+  </button>
+  <button mat-raised-button color="primary" [routerLink]="['edit']">
+    <fa-icon icon="edit"></fa-icon>&nbsp;&nbsp;
+    Edit
+  </button>
+  <button mat-raised-button color="warn" (click)="deleteHoliday()">
+    <fa-icon icon="trash"></fa-icon>&nbsp;&nbsp;
+    Delete
+  </button>
```

Comment:
Please add the condition for the buttons to be displayed as in the community app in form of `ng-show` and `ng-hide`.

Reply:
Only the Activate button will get visible or invisible according to the status right?

Reply:
After activation it should redirect to a page called 'viewMakerCheckerTask' . Is it implemented?

Reply:
> Only the Activate button will get visible or invisible according to the status right?

Yes

Reply:
> After activation it should redirect to a page called 'viewMakerCheckerTask' . Is it implemented?

No doesn't seem so.

Reply:
Okay. Changes added. Please review